DataFormats/MuonReco/interface/Muon.h

```diff
@@ -199,53 +199,54 @@ namespace reco {
     ///
     // When adding new selectors, also update DataFormats/MuonReco/interface/MuonSelectors.h string to enum map
     enum Selector {
-      CutBasedIdLoose = 1UL << 0,
-      CutBasedIdMedium = 1UL << 1,
-      CutBasedIdMediumPrompt = 1UL << 2,  // medium with IP cuts
-      CutBasedIdTight = 1UL << 3,
-      CutBasedIdGlobalHighPt = 1UL << 4,  // high pt muon for Z',W' (better momentum resolution)
-      CutBasedIdTrkHighPt = 1UL << 5,     // high pt muon for boosted Z (better efficiency)
-      PFIsoVeryLoose = 1UL << 6,          // reliso<0.40
-      PFIsoLoose = 1UL << 7,              // reliso<0.25
-      PFIsoMedium = 1UL << 8,             // reliso<0.20
-      PFIsoTight = 1UL << 9,              // reliso<0.15
-      PFIsoVeryTight = 1UL << 10,         // reliso<0.10
-      TkIsoLoose = 1UL << 11,             // reliso<0.10
-      TkIsoTight = 1UL << 12,             // reliso<0.05
-      SoftCutBasedId = 1UL << 13,
-      SoftMvaId = 1UL << 14,
-      MvaLoose = 1UL << 15,
-      MvaMedium = 1UL << 16,
-      MvaTight = 1UL << 17,
-      MiniIsoLoose = 1UL << 18,      // reliso<0.40
-      MiniIsoMedium = 1UL << 19,     // reliso<0.20
-      MiniIsoTight = 1UL << 20,      // reliso<0.10
-      MiniIsoVeryTight = 1UL << 21,  // reliso<0.05
-      TriggerIdLoose = 1UL << 22,    // robust selector for HLT
-      InTimeMuon = 1UL << 23,
-      PFIsoVeryVeryTight = 1UL << 24,  // reliso<0.05
-      MultiIsoLoose = 1UL << 25,       // miniIso with ptRatio and ptRel
-      MultiIsoMedium = 1UL << 26,      // miniIso with ptRatio and ptRel
-      PuppiIsoLoose = 1UL << 27,
-      PuppiIsoMedium = 1UL << 28,
-      PuppiIsoTight = 1UL << 29,
-      MvaVTight = 1UL << 30,
-      MvaVVTight = 1UL << 31,
-      LowPtMvaLoose = 1UL << 32,
-      LowPtMvaMedium = 1UL << 33,
-      MvaIDwpMedium = 1UL << 34,
-      MvaIDwpTight = 1UL << 35,
+      CutBasedIdLoose = 0,
+      CutBasedIdMedium = 1,
+      CutBasedIdMediumPrompt = 2,  // medium with IP cuts
```

Comment:
wouldn't this break user code for those who used explicit values  like `::passed(4)` ?

Reply:
passed(4) would call `bool passed(uint64_t selection)` which looks at the mask properly

Reply:
I'm relatively certain that `passed(4UL)` would use `passed(uint64_t selection)`, but less so about `passed(4)`. Please clarify if you checked it explicitly.

Reply:
I have not indeed,

Reply:
My two cents: I'm not sure if `::pased(4)` will be broken or not, but in the muon POG documentation[1] we recommend (and hence only support) to always access the selectors via the `Selector` format. I don't see the use case of passing explicit values since the `enum` should always be declared when running over MINI(AOD). 

[1] https://twiki.cern.ch/twiki/bin/view/CMS/SWGuideMuonSelection#Muon_selectors_Since_9_4_X